tests/test_watchdog.js

```diff
@@ -0,0 +1,135 @@
+/*******************************************************************************
+ * Copyright 2015 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+if (process.platform !== 'linux') {
+  console.log('1..0 # SKIP watchdog is Linux-only for now');
+  return;
+}
+
+if (process.versions.v8 >= '3.15' && process.versions.v8 < '3.29') {
```

Comment:
also, its not clear to me why this restriction is in the test code, because the c++ doesn't seem to have a similar check, I wonder if its just the tests not stable on these v8 versions.

@bnoordhuis, comments?